src/Microsoft.Health.Fhir.Shared.Core/Features/Resources/Patch/FhirPathPatch/FhirPathPatchBuilder.cs

```diff
@@ -0,0 +1,136 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using FhirPathPatch.Operations;
+using Hl7.Fhir.Model;
+using static Hl7.Fhir.Model.Parameters;
+
+namespace FhirPathPatch
```

Comment:
Just a small one, namespaces don't look consistent with the rest of the project

Reply:
Good catch, thanks. We wrote this as a standalone library and never updated this. I've combed the logic piece of FHIRPatch and cleanup up a bunch of things.